src/mpl/include/mpl_base.h

```diff
@@ -98,6 +98,18 @@
 #define likely(x_)   (x_)
 #endif
 
+#ifdef HAVE_C11__STATIC_ASSERT
+#define MPL_static_assert(cond_,msg_) _Static_assert(cond_,msg_)
```

Comment:
Make sure the commit message reflects the case change we made in the code. `MPL_Static_assert` -> `MPL_static_assert`.